src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -47,42 +47,45 @@ The docs board has the following columns:
     </tr>
     <tr>
       <td>
-        **Hero: to do**
+        **Scheduled work**
       </td>
       <td>
-        PRs that the Hero needs to review, publish, and  follow up with SMEs as needed.  
-        
-        **Hero: Assign yourself as Assignee**.
+        Use this column for "What's new" posts or other time-sensitive issues that you want the hero to watch. Insert the projected publication date in the title so everyone can see it.
       </td>
     </tr>
     <tr>
       <td>
-        **In review (Hero or any TW)**
+        **In progress**
       </td>
       <td>
-        Drag PRs to this column when they are being reviewed. This shows who is reviewing and what is being reviewed, so two writers don’t mistakenly work on the same PR.
+        Drag PRs to this column when you are working on them. Make sure your face is on the PRs so other writers don’t mistakenly work on it.
       </td>
     </tr>
     <tr>
       <td>
-        **Any TW: Writer needs PR review**
+        **Needs review**
       </td>
       <td>
-        PRs from Tech Docs team members that need a light edit pass to make sure everything in GitHub is correct. This should be checked by other writers every few hours so PRs don’t get stale. 
-        
-        If you have a PR that’s been lingering here too long, ask for a reviewer in #doc_sprint_talk. 
-        
-        **Whoever takes it: assign yourself as Reviewer.**
+        This is where you insert your PRs when you are ready to get a review. This is for all types of reviews: everything from minor PRs to PRs related to complicated Jiras.
+
+        1. Make sure you insert comments about what kind of review you need in the body of the PR.
+        2. Either ping a colleague for a review, or let a colleague pick it up at random.
+        3. If you want a specific person to review the PR, add their name as a reviewer.
+        4. If you have a PR that’s been lingering here too long, ask for a reviewer in #doc_sprint_talk.
+
+
       </td>
     </tr>
     <tr>
       <td>
-        **Any TW: needs peer edit**
+        **In review**
       </td>
       <td>
-        Like our Needs Peer Edit column in Jira: A writer has requested a review of their PR. Review their PR in GitHub and leave comments. 
-        
-        **Whoever takes it: assign yourself as Reviewer.**
+        When you pick up a PR from **Needs review**:
+        1. Make sure your name is listed as a reviewer.
+        2. Move the PR into this column while you review it.
+        2. Review the PR in GitHub and leave comments.
+        3. When you are finished with the review, move it to ** Waiting on technical writer to merge**.
```

Comment:
Careful with the **, there's a space before Waiting.

Reply:
@urbiz-nr, good catch! I'll fix this!